acquire/acquire.py

```diff
@@ -1523,6 +1523,35 @@ class OSX(Module):
     ]
 
 
+@register_module("--osx-applications")
+class OSXApplications(Module):
+    DESC = "OS-X application files"
+    SPEC = [
+        # applications
```

Comment:
```suggestion
        # Info.plist from all installed applications
```

Reply:
Added it to the description of the new module